src/utils.js

```diff
@@ -76,6 +76,8 @@ export const generateProjectConfig = (projectId, name) => ({
   name: name,
   id: projectId,
   secret: generateId(),
+  AES: generateId(),
```

Comment:
Replace `AES` everywhere with `aesKey`. Don't use `generateId` to generate the value of `aesKey`. Instead first generate a random 32 bytes string and then base64 encode it